watertap/tools/parameter_sweep/tests/test_parameter_sweep.py

```diff
@@ -1651,9 +1651,14 @@ def _build_header_list_from_csv(csv_filename):
     return csv_header
 
 
-def _read_output_h5(filepath):
+def _read_output_h5(filevar):
 
-    f = h5py.File(filepath, "r")
+    if isinstance(filevar, str):
+        f = h5py.File(filevar, "r")
+    elif isinstance(filevar, h5py._hl.group.Group):
```

Comment:
```suggestion
    elif isinstance(filevar, h5py.Group):
```
Let's use the public alias in case `h5py` is ever reorganized.